himbaechel/himbaechel_api.h

```diff
@@ -102,6 +109,20 @@ struct HimbaechelAPI
     virtual bool isClusterStrict(const CellInfo *cell) const;
     virtual bool getClusterPlacement(ClusterId cluster, BelId root_bel,
                                      std::vector<std::pair<CellInfo *, BelId>> &placement) const;
+
+    // Graphics
+    virtual uint32_t gfxAttributes() { return 0; }
+
+    virtual void gfxTileBel(std::vector<GraphicElement> &g, int x, int y, int z, int w, int h,
```

Comment:
can you perhaps think of some slightly more descriptive names for these API functions, and also add some documentation about them, so I can review them better?

Reply:
sure, will update these, but probably not today